src/Parachain.sol

```diff
@@ -156,7 +160,7 @@ abstract contract Parachain {
         return abi.encodePacked(hex"00", bytes4(_paraId));
     }
 
-    function x1(uint32 _paraId) private pure returns (bytes[] memory) {
+    function x1(uint32 _paraId) public pure returns (bytes[] memory) {
```

Comment:
This shouldnt be public. I think the general pattern for exposing this is via the TestParachain.sol contract which extends Parachain.sol. One can then expose internal functions there rather than making them public.

Reply:
Thanks!